app/models/generic_phrase.rb

```diff
@@ -72,6 +72,17 @@ def self.most_frequent_for_page(page, start_date, end_date)
         .map { |phrase_text, total| { phrase_text: phrase_text, total: total } }
   end
 
+  def first_appeared_at
+    first_survey_mentioning_generic_phrase = Survey
+      .joins(survey_answers: [{ mentions: [{ phrase: :phrase_generic_phrases }] }])
+      .where("phrase_generic_phrases.generic_phrase_id" => id)
+      .order("surveys.started_at asc")
+      .limit(1)
+      .first
+
+    first_survey_mentioning_generic_phrase.started_at
```

Comment:
I actually can't quite figure this out myself - is it possible that there could be no results? If there were none, this would throw `undefined method 'started_at' for nil:NilClass`. I don't think so but just raising it just in case

Reply:
Good question, but there should be no valid case to make this a possibility. If a generic phrase had no results then there's no way it could have been created,

Reply:
If this weren't true then you'd get the exception in this model at line 81 also, when we call `.first`.

Reply:
Thought so